dotcom-rendering/src/components/Badge.tsx

```diff
@@ -23,6 +23,15 @@ const labsSectionBadgeSizingStyles = css`
 	}
 `;
 
+const imageAdvertisingPartnerStyles = css`
+	${from.leftCol} {
+		max-width: 130px;
+	}
+	${from.wide} {
+		max-width: 100%;
+	}
+`;
```

Comment:
A light thought on this declaration: if my assumption is correct, the default width for the label will be 100%, so it might be somewhat more explicit to use the `between.leftCol.and.wide` media query to apply the `130px` max-width. I think this would have the exact same outcome, but it might help future travellers to understand the intent of the design better.